app/scripts/modules/core/serverGroup/serverGroup.component.html

```diff
@@ -0,0 +1,60 @@
+<div class="rollup-pod-server-group clickable clickable-row"
+     waypoint="{{$ctrl.viewModel.waypoint}}"
+     analytics-on="click"
+     analytics-category="Cluster Pod"
+     analytics-event="Load Server Group Details"
+     ng-click="$ctrl.loadDetails($event)"
+     ng-class="{
+        disabled: $ctrl.viewModel.serverGroup.isDisabled,
+        active: $ctrl.$rootScope.$state.includes('**.serverGroup', {region: $ctrl.viewModel.serverGroup.region, accountId: $ctrl.viewModel.serverGroup.account, serverGroup: $ctrl.viewModel.serverGroup.name, provider: $ctrl.viewModel.serverGroup.type})
+     }">
+  <div class="cluster-container">
+    <div class="server-group-title" sticky-header added-offset-height="67"
+         sticky-if="headerIsSticky()">
+      <div class="container-fluid no-padding">
+        <div class="row">
+          <div class="col-md-{{ $ctrl.viewModel.images ? 9 : 8 }} col-sm-6 section-title">
+            <input type="checkbox" ng-if="$ctrl.$scope.sortFilter.multiselect"
```

Comment:
Should be able to attach `sortFilter` directly to `$ctrl` and omit `$scope` (applies in a few places in this template)